policies/intel-google-compute-instance-template-enforce-min-cpu-platform/intel-google-compute-instance-template-enforce-min-cpu-platform.sentinel

```diff
@@ -8,7 +8,7 @@ param valid_actions default [
 ]
 
 doc = {
-	"allowed":   ["Intel Cascade Lake", "Intel Ice Lake", "Intel Sapphire Rapids"],
+	"allowed":   ["Intel Ice Lake", "Intel Sapphire Rapids"],
```

Comment:
This change may be a breaking change. In the policies.md file for the GCP VM - https://github.com/intel/terraform-intel-gcp-vm/blob/main/policies.md, it shows that compute optimized instances like  c2-standard-4, c2-standard-8, c2-standard-16, c2-standard-30, c2-standard-6 are allowed. These instances are based on Cascade lake. So if we remove the Cascade lake from the sentinel policy, then it may break the gcp vm module.

Should we consider making similar changes in the GCP VM module?

Reply:
Trimmed `approved.json` of the mentioned instance values : 

c2-standard-4, c2-standard-8, c2-standard-16, c2-standard-30, c2-standard-60

Reply:
@mandalrajiv Thoughts? Are you ok approving it?

Reply:
@lucasmelogithub - looks like the c2-standard* instances are not trimmed in all places within approved.json file. It is showing up under gcpinstance and gcpinstancetemplate. See below. Should we remove c2-standard* from all occurrences in approved.json?

![image](https://github.com/intel/policy-library-intel-gcp/assets/90872446/d06c8217-0c7b-4f16-a9ab-13a4511b1222)

![image](https://github.com/intel/policy-library-intel-gcp/assets/90872446/ee43a437-7094-4214-b32a-7900e73a6970)